### PR TITLE
28 days of Vim

### PR DESCRIPTION
**Pull request by rogeruiz**
My partner told me that when I started on the journey of learning Vim that I should try to stick to it for 28 days in order to make a habit of it. While I didn't keep meticulous notes at the time, there have been many things I've learned and retained about using Vim and all of the ~~un~~necessary crazy that goes along with it.

** This collection of posts is to help new comers to Vim get interested in making the leap or just learn why everyone who uses it long enough is so passionate about _you_ needing to finally switch over to it.**

##### In other words, Vim just might make you go crazy.

### CODE REVIEW

**Comment by rogeruiz** [comment]
There are some asset pipeline fixes in this PR as well, boo hoo. :sob: Oh well. I'm going to try to keep this focused on the Vim posts from now on though.
